conf/modules.config

```diff
@@ -25,13 +25,15 @@ process {
     withName: SAMPLESHEET_CHECK {
         publishDir = [
             path: { "${params.outdir}/pipeline_info" },
-            mode: 'copy',
+            mode: params.publish_dir_mode,
+            saveAs: { filename -> filename.equals('versions.yml') ? null : filename }
         ]
     }
 
     withName: CUSTOM_DUMPSOFTWAREVERSIONS {
         publishDir = [
             path: { "${params.outdir}/pipeline_info" },
+            mode: params.publish_dir_mode,
             pattern: '*_versions.yml'
         ]
     }
```

Comment:
I am a little surprised that the code below, e.g.
```
    withName: CHECK_REQUESTED_MODELS {
        publishDir = [
            path: { "${params.outdir}/reports" },
        ]
        ext.args = "--tools ${params.tools} --max_length ${params.max_peptide_length} --min_length ${params.min_peptide_length} "
    }
```
does not break, the `path: { "${params.outdir}/reports" },` should not end with a comma, I thought (but might be wrong?)
Also, I thought `mode: params.publish_dir_mode` is needed in all modules where something is actually copied into the results folder.